### DISCUSSION TITLE
Switch user / One-Time Password

### DISCUSSION THREAD

**Discussion by LfG-B** [rustdesk/rustdesk#5402] [General]
### Bug Description

It's not possible to switch user if you start a session with one-time password or without password (remote user allows acess manually)

### How to Reproduce

1. Start session with one-time-password only (with no permanent password set).
2. Go on remote computer to "Windows start" => User => Switch user. Session is interrupted, local logon screen is not accessible, no logon on remote is possible

### Expected Behavior

Like TeamViewer: local logon screen is seen and logon as another user (f.e. as local administrator) is possible. I believe (not sure) Teamviewer service "intercepts" a session control to acess UAC in this case.

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.2.1 -> 1.2.1

### Screenshots

![RustDeskScreen1](https://github.com/rustdesk/rustdesk/assets/142325349/bbf6b75c-94f2-460c-8563-b77e3cf97c74)
![RustDeskScreen2](https://github.com/rustdesk/rustdesk/assets/142325349/5ab1766c-f537-44e0-974b-fb3e434e0bef)

### Additional Context

We are not allowed to work with permanent passwords because of our internal security policys. Thank you for implementation of one-time passwords!

Workaround I found: send "Insert lock". So local logon screen is still acessible from remote. After you log on as local admin, user on other side should "Allow" session manually. This is aceptable as workaround, but not fully safe - you loose control of remote computer if user not allows this connection. And user on remote side is able to act as local admin further: local admin is left logged on on remote computer!

**Comment by elhananjair** [comment]
Is there any solution for this feature?
